apps/OboeTester/docs/TestOutput.md

```diff
@@ -14,7 +14,7 @@ DRAFT for testing image embedding.
       The resulting setting will displayed on the far right when the stream is opened.<br/>
       API: select between OpenSL ES or AAudio (default)<br/>
       Device: setect output device by type.<br/>
-      Format: note that the 24 and 32-bit formats are only supported in Android 12+<br/>
+      Format: note that the 24 and 32-bit formats are only supported in Android 12+. MP3 is only supported on Android 16+. It uses a 44.1kHz stereo MP3 file for playback. The stream must be configured as 44.1kHz stereo selecting MP3 format.<br/>
```

Comment:
`It uses a 44.1kHz stereo MP3 file for playback.` should be `The test uses a 44.1kHz stereo MP3 file for playback.` Otherwise, it implies that other file types won't work